packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -281,6 +280,15 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 	constructor( config: EditorConfig = {} ) {
 		super();
 
+		if ( 'sanitizeHtml' in config ) {
+			/**
+			 * Configuration property `config.sanitizeHtml()` was removed in CKEditor version 43.1.0 and is no longer supported.
```

Comment:
I'd add a link to [https://ckeditor.com/docs/ckeditor5/latest/updating/guides/update-to-43.html](https://ckeditor.com/docs/ckeditor5/latest/updating/guides/update-to-43.html) and a short explanation there what happened and why.